### DISCUSSION TITLE
Detect circles with a min/max radius in an image using Hough Transform

### DISCUSSION THREAD

**Discussion by smillerk2** [SixLabors/ImageSharp#2445] [Q&A] [answered]
How can I use ImageSharp to apply Hough Transform to detect circles that match a given radius in an image? I need to get the X, Y and radius coordinates of all detected circles so that I can then crop the found images.

Has anyone done this or have any pointers?

**Comment by antonfirsov** [comment, marked as answer]
Computer Vision algorithms are out of scope for ImageSharp, you would need to use another library like EmguCV or opencvsharp for this.

**Reply by smillerk2** [reply]
Appreciate the quick response. I have already looked at EmguCV but due to dependencies on cvextern.dll it is proving difficult to bundle and run on a non windows environment. I'll take a look at opencvsharp.
